packages/react-storage/src/components/StorageBrowser/composables/DataTable/headers/SortHeader.tsx

```diff
@@ -23,7 +21,7 @@ export const SortHeader = ({ content }: SortHeaderProps): React.JSX.Element => {
       className={`${STORAGE_BROWSER_BLOCK_TO_BE_UPDATED}__table-sort-header`}
       onClick={onSort}
     >
-      {label && capitalize(label)}
```

Comment:
why are we removing capitalization?

Reply:
Capitalization is handled by the display text API